syntheseus/tests/search/graph/test_standardization.py

```diff
@@ -1 +1,27 @@
-"""Graph standardization is not tested at this time, but should be tested in the future."""
+"""
+At the moment, only a smoke-test is done for graph standardization,
+but the exact behaviour is not well-tested.
+These tests should be added in the future.
+"""
+
+import pytest
+
+from syntheseus.search.graph.and_or import AndOrGraph
+from syntheseus.search.graph.molset import MolSetGraph
+from syntheseus.search.graph.standardization import get_unique_node_andor_graph
+
+
+def test_smoke_andor(
+    andor_graph_non_minimal: AndOrGraph,
+):
+    with pytest.warns(UserWarning):
+        output = get_unique_node_andor_graph(andor_graph_non_minimal)
+
+    assert len(output) == len(andor_graph_non_minimal)  # no nodes deleted here
+
+
+def test_smoke_molset(
+    molset_tree_non_minimal: MolSetGraph,
+):
+    with pytest.warns(UserWarning):
+        get_unique_node_andor_graph(molset_tree_non_minimal)
```

Comment:
Just noting that these function signatures would normally fit on a single line, and the reason why `black` doesn't reformat this is due to the [magic-trailing comma behaviour](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) (it interprets the comma in the argument list as a strong preference against having the signature on a single line). I'm assuming it was not your intention to trigger this behaviour though 🙂
```suggestion
def test_smoke_andor(andor_graph_non_minimal: AndOrGraph):
    with pytest.warns(UserWarning):
        output = get_unique_node_andor_graph(andor_graph_non_minimal)

    assert len(output) == len(andor_graph_non_minimal)  # no nodes deleted here


def test_smoke_molset(molset_tree_non_minimal: MolSetGraph):
    with pytest.warns(UserWarning):
        get_unique_node_andor_graph(molset_tree_non_minimal)
```

Reply:
Good point, this was not intentional. I just fixed it.